### PR TITLE
ex_2a modify lab.txt

### PR DESCRIPTION
**Pull request by raluca114**

### CODE REVIEW

File: lab.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 ex 1 for lab
+ex 2a for lab
```
Comment (annelorecomnoiu) [inline lab.txt:2]: please  add ";" at the end of the line

**Review by annelorecomnoiu — CHANGES_REQUESTED** [review]
update

File: lab.txt (hunk 1 above)
Comment (raluca114) [reply lab.txt:2]: te pup

Comment (annelorecomnoiu) [reply lab.txt:2]: sho

Comment (raluca114) [reply lab.txt:2]: ms apreciez

**Pull request merged**